ere-guests/empty-program/risc0/src/main.rs

```diff
@@ -0,0 +1,8 @@
+//! Risc0 guest program
+
+extern crate alloc;
+
+use risc0_zkvm::guest::env;
```

Comment:
Found something very weird here. This way the empty-program works, but this is an unused import. If I replace with `risc0_zkvm::guest::entry!(main);` it says that the main entry is already defined. If I remove it complains the program has no entry point. :shrug: